lib/web/apiserver_test.go

```diff
@@ -1035,6 +1041,35 @@ func (s *WebSuite) TestResizeTerminal(c *C) {
 	}
 }
 
+func (s *WebSuite) TestTerminalPing(c *C) {
```

Comment:
Any chance we could write this without gocheck?

Reply:
While possible, it does require that we severely rearchitect how all web tests work and I don't feel like that this is in the scope of this PR.

Reply:
Fair enough.